### PR TITLE
Meeting 064.md

### PR DESCRIPTION
**Pull request by ShubhangiSG**

### CODE REVIEW

**Comment by poojaranjan** [comment]
Please update README.

File: All EIPIP Meetings/Meeting 064.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+
+**Sam**
+See you and thanks for hosting.
+
+Everyone wishes goodbye. 
+
+---------------------------------------
+# Attendees
+
+Pooja Ranjan
```
Comment (poojaranjan) [inline All EIPIP Meetings/Meeting 064.md:295]:
```suggestion
- Pooja Ranjan

```

File: All EIPIP Meetings/Meeting 064.md [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+Sam Wilson
```
Comment (poojaranjan) [inline All EIPIP Meetings/Meeting 064.md:296]:
```suggestion
- Sam Wilson

```

File: All EIPIP Meetings/Meeting 064.md [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+Jose
```
Comment (poojaranjan) [inline All EIPIP Meetings/Meeting 064.md:297]:
```suggestion
- Jose

```

File: All EIPIP Meetings/Meeting 064.md [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+Lightclient
```
Comment (poojaranjan) [inline All EIPIP Meetings/Meeting 064.md:298]:
```suggestion
- Lightclient

```

File: All EIPIP Meetings/Meeting 064.md [hunk 5]
```diff
@@ -0,0 +1,304 @@
...
+Felix
```
Comment (poojaranjan) [inline All EIPIP Meetings/Meeting 064.md:299]:
```suggestion
- Felix

```

**Review by poojaranjan — COMMENTED** [review]
Please add Summary with Decisions/Action Items

**Comment by ShubhangiSG** [comment]
> Please update README.

Updated README - https://github.com/ethereum-cat-herders/EIPIP/pull/179

**Comment by ShubhangiSG** [comment]
> Please add Summary with Decisions/Action Items

Added Decision Items

File: All EIPIP Meetings/Meeting 064.md [hunk 6]
```diff
@@ -0,0 +1,331 @@
...
+**Pooja**
+Yeah one other thing that I would like to make a mention of generally in case of any changes to EIP-1 earlier we were having like two editors’ approval, but now we have like around five editors available, so it will be nice to have minimum three editors on board and get that approval done. I'm not sure if this is already added in the bot and it is taken care of, but if not, I think that would be a good idea.
+
+**Lightclient**
+There's also like a general courtesy I think that we haven't merged too many things in the past where editors disagreed even if we had two editors, unless the editor who disagreed said I’m not going to block this, like there is, does seem to be a bit of veto power amongst single editors if they think a change is bad, I don't want to like encode that but I think that it's like a reasonable courtesy to not just merge things after two, even if somebody has already said they're pretty against it.
+
+**Sam**
+Yeah, that was entirely my bad, like it's entirely my fault, like Pandapip has been adding EIPs to a manual merge-like list because some of them require it and I didn't realize that we didn't have consensus on this one so that's entirely my bad.
```
Comment (Pandapip1) [inline All EIPIP Meetings/Meeting 064.md:93]: FYI @SamWilsn - we *did* have consensus on it on EIPIP meeting 62. It's just that 3 of the editors were absetnt, a fact that clearly both of us forgot.

**Pull request merged**